Pathfinder.elm

```diff
@@ -47,8 +50,12 @@ obstacleTiles obstacles =
   |> List.concat
 
 
-find : (Int, Int) -> List (Int, Int) -> (Int, Int) -> (Int, Int) -> List (Int, Int)
-find = astar
+createMap : List (Int, Int) -> (Int, Int) -> Map
+createMap = Astar.createGrid
+
+
+find : (Int, Int) -> (Int, Int) -> Map -> List (Int, Int)
+find = Astar.findPath
```

Comment:
i think it would be better to have astar.elm separately, because it is now a module completely independent from rendering.